raml-parser-interface/src/main/java/org/mule/apikit/model/api/ApiReference.java

```diff
@@ -50,23 +49,20 @@ static ApiReference create(final URI uri) {
 
   String getLocation();
 
-  String getFormat();
+  ApiFormat getFormat();
```

Comment:
This can't  be changed

Reply:
My bad, didn't realize that is under api package. Reverted